tests/components/tomorrowio/test_config_flow.py

```diff
@@ -0,0 +1,292 @@
+"""Test the Tomorrow.io config flow."""
+from unittest.mock import patch
+
+import pytest
+from pytomorrowio.exceptions import (
+    CantConnectException,
+    InvalidAPIKeyException,
+    RateLimitedException,
+    UnknownException,
+)
+
+from homeassistant import data_entry_flow
+from homeassistant.components.climacell.const import DOMAIN as CC_DOMAIN
+from homeassistant.components.tomorrowio.config_flow import (
+    _get_config_schema,
+    _get_unique_id,
+)
+from homeassistant.components.tomorrowio.const import (
+    CONF_TIMESTEP,
+    DEFAULT_NAME,
+    DEFAULT_TIMESTEP,
+    DOMAIN,
+)
+from homeassistant.config_entries import SOURCE_IMPORT, SOURCE_USER, ConfigEntryState
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_API_VERSION,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+    CONF_RADIUS,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+
+from .const import API_KEY, MIN_CONFIG
+
+from tests.common import MockConfigEntry
+from tests.components.climacell.const import API_V3_ENTRY_DATA
+
+
+@pytest.fixture(name="tomorrowio_entry_skip_setup", autouse=True)
+def tomorrowio_entry_skip_setup_fixture():
```

Comment:
Did we plan to patch the integration setup function?

Reply:
changing the approach and just using the library patch everywhere